otcdocstheme/theme/otcdocs/header.html

```diff
@@ -81,36 +81,36 @@
         <scale-telekom-nav-flyout hover>
           <scale-telekom-mega-menu>
             <scale-telekom-mega-menu-column>
-              <a href="{{base_url}}/blueprints/" slot="heading">Plan</a>
+              <a href="{{base_url}}/caf/plan/index.html" slot="heading">Plan</a>
```

Comment:
Are you sure that his works? Because you forgot the `/architecture-center` in front here.

Reply:
Ah yes, that needs /architecture-center indeed!